hera_pspec/pspecdata.py

```diff
@@ -3956,7 +4064,9 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None,
             uvp.cov_array_real = cov_array_real
             uvp.cov_array_imag = cov_array_imag
             uvp.cov_model = cov_model
-
+        if store_QE_error:
+            uvp.stats_array = odict()
+            uvp.stats_array["QE_"+cov_model] = stats_array_cov_model
```

Comment:
likewise can we rename this to `cov_model` + "_diag"

Reply:
Also, in this indented if statement, we should also remove the `cov_array_real`, as the point of storing its diagonal is so we don't have to store the full array

Reply:
Since choices of store_cov and store_cov_diag are independent to each other, we can just set store_cov=False and store_cov_diag=True to not store the full cov_array.

Reply:
I think my comment above applies here: I think we should store either the full cov_array as cov_array_real etc. or just its diagonal in stats_array.

I recall us having this discussion a few years ago (when you initially made the cov_array_real and cov_array_imag distinction), and we decided to not store the diagonal b/c it can be accessed by just manually taking the diagonal of cov_array. The same logic applies here I think--if someone is explicitly asking for `store_cov_diagonal`, then we should remove the full array, as the only reason we would just store the diagonal is to save memory / disk space.

Reply:
Maybe we just set store_cov=False when store_cov_diag=True?

Reply:
Ah, yes I think that would work!